src/main/resources/jupyter/google_sign_in.js

```diff
@@ -13,25 +13,30 @@ require.config({
 // Leonardo has logic to find/replace templated values in the format $(...).
 // This will be replaced with the real Google Client ID before uploading to the notebook server.
 var clientId = $(googleClientId);
+var loginHint = $(googleLoginHint);
```

Comment:
The loginHint is key to avoiding a pop-up. It is substituted with the user email when the cluster is created.